integration/benchmark/server/bench.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -x
+set -o pipefail
+
+# this flow-go sub folder will be where all the TPS tests will be run
+# this will keep the TPS automation code separate from the code that's being tested so we won't run into issues
+# of having old versions of automation code just because we happen to be testing an older version flow-go
+git clone https://github.com/onflow/flow-go.git
+cd flow-go/integration/localnet
+
+git fetch
+git fetch --tags
+
+while read -r branch_hash; do
+    hash="${branch_hash##*:}"
+    branch="${branch_hash%%:*}"
+
+    git checkout "$branch" || continue
+    git reset --hard "$hash"  || continue
+
+    git log --oneline | head -1
+    git describe
+
+    make -C ../.. crypto_setup_gopath
+
+    # instead of running "make stop" which uses docker-compose for a lot of older versions,
+    # we explicitly run the command here with "docker compose"
+    DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker compose -f docker-compose.metrics.yml -f docker-compose.nodes.yml down -v --remove-orphans
```

Comment:
It doesn't look like `docker-compose.metrics.yml` is ever spun up in this automation. Would it be safe to remove the `-f docker-compose.metrics.yml`?

Reply:
Nice catch. Removed

891cccc60118b824957f1062589a5cfc5a98948d